api/MIDIInput.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "MIDIInput": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MIDIInput",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "43"
+          },
+          "chrome_android": {
+            "version_added": false
```

Comment:
Both versions of Android should say `"43"`.

Reply:
👍